### PR TITLE
Switch deployment providers from Vercel to Netlify 🎉

### LINKED ISSUE #697: Netlify: Migrate from Vercel to Netlify
### What's your idea?

https://docs.netlify.com/welcome/add-new-site/

- [x] Transfer environment variables from [Vercel](https://vercel.com/ethanthatonekid/acm-csuf-site/settings/environment-variables) to [Netlify](https://app.netlify.com/sites/acmcsuf/settings/env).
- [ ] Set up [Netlify Deploy Previews](https://docs.netlify.com/site-deploys/deploy-previews/).
- [ ] Set up `acmcsuf.com` domain configuration ([Configure external DNS for a custom domain](https://docs.netlify.com/domains-https/custom-domains/configure-external-dns/)).
- [x] Replace [Vercel Analytics](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/src/lib/public/analytics/vitals.ts#L28) ([current implementation](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/src/routes/+layout.svelte#L10)) with [Netlify Analytics](https://docs.netlify.com/monitor-sites/analytics/) ([Lighthouse](https://docs.netlify.com/monitor-sites/lighthouse/), [related](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/svelte.config.js#L12-L13)).
- [x] Update [.env.example](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/.env.example#L6).
- [ ] Update documentation ([README.md](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/README.md?plain=1#L21) and [GETTING_STARTED.md](https://github.com/EthanThatOneKid/acmcsuf.com/blob/7e328c6cd9606fc394b3f0e3c0072550ad72849b/GETTING_STARTED.md?plain=1#L7)).

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Resolves #697.

### CODE REVIEW

**Comment by EthanThatOneKid** [comment]
As part of this PR, I transferred the required environment variables to [Netlify](https://app.netlify.com/sites/acmcsuf/settings/env).
image

**Comment by EthanThatOneKid** [comment]
As part of this PR, I gave the keys to [Netlify](https://app.netlify.com/teams/ethanthatonekid/dns/acmcsuf.com) for handling the `acmcsuf.com` domain.
image

File: package.json [hunk 1]
```diff
@@ -29,16 +29,18 @@
   "devDependencies": {
     "@playwright/test": "^1.27.1",
     "@size-limit/file": "^8.1.0",
-    "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/adapter-netlify": "^1.0.0-next.84",
+    "@sveltejs/kit": "^1.0.0-next.544",
```
Comment (charliettaylor) [inline package.json]: Is there a specific reason these two are no longer `next`?

**Review by anhduy1202 — APPROVED** [review]
lgtm

**Pull request merged**